app/main.py

```diff
@@ -1 +1,46 @@
-#  write your code here
+from typing import Any
+
+
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int, is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> Any:
```

Comment:
The return type of the `feed` method is declared as `Any`, but it actually returns an `int`. Consider changing the return type to `int` for clarity and accuracy.